addons/stock/views/stock_picking_type_views.xml

```diff
@@ -204,9 +204,12 @@
                                     </div>
                                 </div>
                                 <div class="col-6 o_kanban_card_manage_section o_kanban_manage_new">
-                                    <h5 role="menuitem" class="o_kanban_card_manage_title">
-                                        <a name="%(action_picking_form)d" type="action">New</a>
-                                    </h5>
+                                    <div role="menuitem">
+                                        <a name="%(action_picking_form)d" type="action">New Transfer</a>
+                                    </div>
+                                    <div role="menuitem">
+                                        <a name="get_action_picking_type_operations" type="object">Product Moves</a>
+                                    </div>
```

Comment:
I'm not sure it's wanted. Are you sure it's not a mistake in rebase?

Reply:
No, it's not. What we have now is that "Prepare Waves" button in the kanban card menu opens a tree view of the moves of the picking type, which doesn't make much sense. So, when I asked THD about it, he told me to keep it but to rename it into "Product Moves" and then change the functionality of "Prepare Wave" button to do the expected (open a wizard with a list view of the move lines to choose and construct a wave).